home/home_3.1/unittest.swift

```diff
@@ -0,0 +1,65 @@
+import XCTest
+
+func calculateValue(x: Double, y: Double, z: Double) -> Double {
```

Comment:
I hope that this function can be moved to a separate file or something to be reused from both test target and the original main program